Documentation/api-reference.md

```diff
@@ -320,6 +323,73 @@ function onAdbmobileApiResponse() as void
 
 ---
 
+### createMediaSession
+
+Call this function to start a new Media session with the given XDM data. The XDM data must be the type of "media.sessionStart". If the "playerName", "channel", and "appVersion" are not provided in the XDM data, the SDK will use the global values passed via "updateConfiguration" API.
```

Comment:
nit:
Creates a new Media session with the provided XDM data. The XDM data event type should be "media.sessionStart". If the `playerName`, `channel`, and `appVersion` are not provided in the XDM data, the SDK will use the global values passed via `updateConfiguration` API.